### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by tensae-b**
# comment 

Blog page and related pages changed to svelte

### CODE REVIEW

File: src/lib/Blog/BlogCard.svelte [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    width: 100%;
+    max-width: 40%;
+    /* height: 274px; */
+    border-radius: 0px 10px 10px 0px;
+  }
+  .left {
+    display: flex;
+    flex-direction: column;
+    padding: 26px 13px;
+    word-wrap: break-word;
```
Comment (Dave-lab12) [inline src/lib/Blog/BlogCard.svelte:58]: Can we use tailwind for this page

File: src/lib/Blog/Hero.svelte [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	}
+	.hero__text {
+		display: flex;
+		flex-direction: column;
+		gap: 14px;
+	}
+	.hero__description {
+		font-weight: 400;
+		font-size: 18px;
+		line-height: 34px;
```
Comment (Dave-lab12) [inline src/lib/Blog/Hero.svelte:64]: Same here too tailwind?

File: src/lib/Blog/BlogCard.svelte (hunk 1 above)
Comment (tensae-b) [reply src/lib/Blog/BlogCard.svelte]: done

File: src/lib/Blog/Hero.svelte [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<section class="hero flex justify-evenly my-24">
+	<div class="hero__background"></div>
+	<div class="hero__background__decoration"></div>
+	<div class="hero__content flex flex-col w-1/2 mr-36 gap-8">
+		<h1 class="hero__title text-4xl font-bold">{title}</h1>
+		<div class="hero__description">
+			<p>{@html description}</p>
+		</div>
+		<div class="hero__cta max-w-36">
+			<GlowingBtn href={url} text="Read more" fontSize="text-lg" />
```
Comment (Dave-lab12) [inline src/lib/Blog/Hero.svelte:21]: We don't need the class names if we changed it to tailwind

**Review by Dave-lab12 — APPROVED** [review]
LFG

**Pull request merged**